packages/server/utils/stripe/StripeManager.ts

```diff
@@ -121,7 +144,9 @@ export default class StripeManager {
   }
 
   async retrieveInvoice(invoiceId: string) {
-    return this.stripe.invoices.retrieve(invoiceId)
+    return this.stripe.invoices.retrieve(invoiceId, {
+      expand: ['payment_intent']
```

Comment:
This is required so that we get the `paymentIntent.status` in `stripeFailPayment`. The payment is considered a failure until the user authorises it.